add-project-script missing in (dev) dependencies?

## Describe the bug

`manage.py initial_sockpuppet` throws error during tutorial [https://sockpuppet.argpar.se/setup-django](https://sockpuppet.argpar.se/setup-django)

## To Reproduce

Just follow the instructions on https://sockpuppet.argpar.se/setup-django

## Expected behavior

There should be no errors

## Screenshots or reproduction
```
./manage.py initial_sockpuppet
Wrote to /home/christian/Projekte/reflex/package.json:

{
  "name": "reflex",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "Christian González <...@...at>",
  "license": "ISC"
}


/home/christian/.nvm/versions/node/v13.6.0/bin/add-project-script -> /home/christian/.nvm/versions/node/v13.6.0/lib/node_modules/add-project-script/dist/index.js
+ add-project-script@1.0.2
added 11 packages from 6 contributors in 1.014s
/bin/sh: Zeile 1: add-project-script: Kommando nicht gefunden.
Build and watch already in package.json, so skipping these
removed 11 packages in 0.199s
```

Maybe [add-project-script](https://www.npmjs.com/package/add-project-script) is missing and should be installed before (globally?) - this should be mentioned in the setup.

## Versions

### Django-sockpuppet
- pypi: 0.5.0

### External tools

- Python: [e.g. 3.9]
- Django: [e.g. 3.5.1]
- OS: [Linux]

Hm, it's not as easy as I thought. There might be a bug/enhancement in django-suckpuppet as well, so maybe I'd like to reopen this bug, If it's ok for you.
the script seems to have problems installing `add-project-script` globally with npm. I tried it with more a nvm environments, and the system node package, no success. I always get the error that it is already installed, and can't overwrite it.

    npm ERR! EEXIST: file already exists, symlink '../lib/node_modules/add-project-script/dist/index.js' -> 
    '/home/christian/.nvm/versions/node/v13.6.0/bin/add-project-script'.

I think the problem is that the script assumes that add-project-script is **not** installed. And if it is, if fails. a `--force` flag to npm (like it suggests it) helps and fixes my problem. 
May I suggest to add this to the script?

--- a/sockpuppet/management/commands/initial_sockpuppet.py
+++ b/sockpuppet/management/commands/initial_sockpuppet.py
@@ -11,7 +11,7 @@ class Command(BaseGenerateCommand):
 
     def handle(self, *args, **options):
         init = 'npm init -y'
-        install = 'npm install -g add-project-script'
+        install = 'npm install -g --force add-project-script'
         subprocess.check_call(init, shell=True)
         subprocess.check_call(install, shell=True)
         try:

